Sprint-2/3-mandatory-implement/1-bmi.js

```diff
@@ -15,5 +15,10 @@
 // It should return their Body Mass Index to 1 decimal place
 
 function calculateBMI(weight, height) {
-    // return the BMI of someone based off their weight and height
-}
\ No newline at end of file
+  const result = +(weight / (height * height)).toFixed(1);
+  return result;
```

Comment:
This works, but I've noticed you're making a variable here that you're only using once.

That can be useful, because the name could be clear for what it is, but I'm not sure the name "result" is very clarifying.

You could instead simplify this down into one line:

```suggestion
  return +(weight / (height * height)).toFixed(1);
```